### PR TITLE
another breaking change to door state topic

### PR DESCRIPTION
**Pull request by ccutrer**

### CODE REVIEW

**Comment by ccutrer** [comment]
hold off on this, it's not going to work

**Comment by ccutrer** [comment]
maybe it is. I just have one door that apparently the magnet is off a bit, and it goes to `closed` for a second when closing, but then bounces to `ajar`, which was previously ignored but now is treated as `OPEN`

**Comment by ktgeek** [comment]
I just put it in place.  I'll go open and close the door a few times :)

File: HOWTOs/configuring_hiome_with_openhab.md [hunk 1]
```diff
@@ -25,12 +25,17 @@ Do not turn on TLS/SSL.
...
-1. Ensure you have the JSONPath Transformation installed.
-2. In Inbox in PaperUI manually add a Generic MQTT Thing and select the Hiome MQTT Broker as the bridge.
-3. In the Generic MQTT Thing you created click the `+` button to add a channel
-4. For "Channel Type" select `Contact`
-5. Provide a unique to your open hab Channel Id (if this is your first channel, you can just use 1.)
-6. Under MQTT state topic you will need to provide the MQTT path to the Hiome. `hiome/1/sensor/<sensor_id>:door`. The sensor id can be found on a sticker on top of your Hiome sensor, or by going to http://hiome.local/api/1/sensors.
-7. Set "Custom On/Open value" to "1"
-8. Set "Custom Off/Closed value" to "0"
-9. Open the "Show More" and put `JSONPATH:$.val` in Incoming value transformations.
+1. Ensure you have the Javascript Transformation installed.
```
Comment (ktgeek) [inline HOWTOs/configuring_hiome_with_openhab.md:28]: I missed this part :(  Once I figured it out, stuff was all good.  That's my fault for reconfiguring mid-stride.

**Comment by ktgeek** [comment]
Oh hey, I wonder if the MAP transformation might have been easier.  But this works too.

**Comment by ccutrer** [comment]
I tried a MAP. I was unable to get a JSONPATH and then a MAP transformation to properly chain. Supposedly you can, but I couldn't get it to work.

**Comment by ktgeek** [comment]
oh DUH, forgot about the JSONPATH part.  Totally makes sense

**Pull request merged**